src/components/Shared/Filters/FilterListItemMultiselect.tsx

```diff
@@ -62,7 +63,7 @@ export const FilterListItemMultiselect: React.FC<
     statusArray.find((status) => status.id === selectedStatus)?.phase;
 
   const groupByPhase = (option: string) =>
-    getLocalizedPhase(t, getStatusPhaseId(option));
+    getLocalizedPhase(getStatusPhaseId(option));
```

Comment:
What are your thoughts to adding brackets to these functions? This one and the 2 above it. I know Bill prefers this and it does make it easier to read. I also know it takes up more room.